test/extended/apiserver/graceful_termination.go

```diff
@@ -96,7 +96,7 @@ var _ = g.Describe("[sig-api-machinery][Feature:APIServer][Late]", func() {
 			}
 		}
 		if len(finalMessageBuilder.String()) > 0 {
-			g.GinkgoT().Errorf("The following API Servers weren't gracefully terminated: %v", finalMessageBuilder.String())
+			result.Flakef("The following API Servers weren't gracefully terminated. Probably kubelet or CRI-O is not giving the time to cleanly shut down. This can lead to connection refused and network I/O timeout errors in other components.\n\n%s", finalMessageBuilder.String())
```

Comment:
are we going to let the node team know that they should take a look and propose a fix ? or is this an issue with the test ?

Reply:
I think the test fine, just that the kubelet cannot ensure the gracefully termination during the node restart. I will ask the node team for a confirmation. Btw, I have copy pasted the new error message from the test above.

Reply:
I don't mind tagging this test as flaky as long as the node team is aware of the issue and is willing to fix it in the future.

Reply:
my understanding here is we can't ensure an ordering without graceful node shutdown, which has other issues. So I agree this is something we're looking towards fixing long term, but it's not something we imminently are working on. how does that sound @p0lyn0mial ? cc @rphillips